src/view/retailer/boostList/filterBooster.tsx

```diff
@@ -0,0 +1,81 @@
+import { useEffect, useMemo, useState } from 'react'
+import { useSelector } from 'react-redux'
+
+import { Checkbox, Col, Row, Space, Typography, Switch } from 'antd'
+
+import { useOwnBoosters } from 'hooks/boosters/useOwnBoosters'
+import { AppState } from 'model'
+import useAmountVoucher from 'hooks/boosters/useAmountVoucher'
+
+type FilterBoosterProps = {
+  onChange: (boosterAddresses: string[]) => void
+}
+
+const FilterBooster = ({ onChange }: FilterBoosterProps) => {
+  const [isHideExpired, setIsHideExpired] = useState(true)
+  const [isHideFreeze, setIsHideFreeze] = useState(true)
```

Comment:
using 'is...' for boolean is a good idea. But in with hidden, I think it better to use hiddenExpired, hiddenFrozen